src/mgr/DaemonServer.cc

```diff
@@ -837,6 +837,21 @@ void DaemonServer::log_access_denied(
         "#client-authentication";
 }
 
+static void print_value(Formatter *f, bufferlist &odata,
+                        const string &name, const string &value)
+{
+  if (f) {
+    f->open_object_section("config");
+    f->dump_string("name", name);
+    f->dump_string("value", value);
```

Comment:
include `source`, `overrides` as below in vanilla `config show <entity>`. I think `default `would also be valuable.